pkg/segment/query/processor/statscommand.go

```diff
@@ -18,45 +18,52 @@
 package processor
 
 import (
+	"fmt"
 	"io"
 
+	"github.com/siglens/siglens/pkg/config"
 	"github.com/siglens/siglens/pkg/segment/query/iqr"
 	"github.com/siglens/siglens/pkg/segment/results/segresults"
+	"github.com/siglens/siglens/pkg/segment/search"
 	"github.com/siglens/siglens/pkg/segment/structs"
 	"github.com/siglens/siglens/pkg/segment/utils"
+	"github.com/siglens/siglens/pkg/segment/writer/stats"
 	toputils "github.com/siglens/siglens/pkg/utils"
 	log "github.com/sirupsen/logrus"
+	bbp "github.com/valyala/bytebufferpool"
 )
 
 type ErrorData struct {
-	// columnName -> error. Tracks errors while reading the column through iqr.Record.ReadColumn()
-	// This is logged as a warning. As the column may not exist in one segment but exist in another.
-	readColumns map[string]error
+	readColumns           map[string]error       // columnName -> error. Tracks errors while reading the column through iqr.Record.ReadColumn
+	cValueGetStringErr    map[string]interface{} // columnName -> error. Tracks errors while converting CValue to string
```

Comment:
You're saying the value is `error`; why do you map to a `interface{}`?

Reply:
Yes, I changed it. It will be updated in my next commit.